### ISSUE TITLE
[Accessibility]: legends in Gauge Chart component are not hoverable and not dispalying fully when text is  long.

### ISSUE THREAD

**Issue by RakshaFauzdar** [carbon-design-system/carbon-charts#1945]
### Application/Team

Vajra

### Environment

Browser

### What happened? What did you expect to happen instead?

In legend-item class inside Guage chart,  legends are not hoverable also text is coming ... after a specific width , I have tried to modify it , still not working. 
I think adding title in **p**  tag inside legend-item will resolve this issue.
Also in other carbon charts components this behavious is not seen.
Here is the link of code behaviour:
value :"MICROSOFT SQL SERVER" is not readable.
https://stackblitz.com/edit/qp7gtter?file=options.js,data.js

### What WCAG 2.1 checkpoint does the issue violate?

AVT3

### Version

@carbon/charts-react": ^1.15.6

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P2
